beacon-chain/monitor/service.go

```diff
@@ -0,0 +1,34 @@
+package monitor
+
+import (
+	types "github.com/prysmaticlabs/eth2-types"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	log = logrus.WithField("prefix", "monitor")
+)
+
+// ValidatorMonitorConfig contains the list of validator indices that the
+// monitor service tracks, as well as the event feed notifier that the
+// monitor needs to subscribe.
+type ValidatorMonitorConfig struct {
+	TrackedValidators []types.ValidatorIndex
```

Comment:
Why not a map? That way you can get rid of the for loop over the entire slice every time you call TrackedIndex

Reply:
+1